libs/logging/src/log_event_ids.ts

```diff
@@ -1,36 +1,34 @@
+/**
```

Comment:
This file was generated by libs/logging/scripts/generate_types_from_toml.ts

Some diffs are just things moving around so I tried to highlight the main ones with comments